README.md

```diff
@@ -53,7 +53,12 @@ const wertWidget = new WertWidget(options);
+| **isCryptoHidden**                                                                                                                                                                                                                               |           optional           | *Boolean* |         `false`          |                                       `true, false`                                       | Allows to hide crypto mentions and exchange rate for NFT purchases if it is enabled for your partnerId.                                                                                                                                                                                                                               |
+| **full_name**                                                                                                                                                                                                                                    |           optional           | *String*  |                          |                        min 3, max 69 letters; RegExp(/(\w+\s)\w+/)                        | User's first and last name                                                                                                                                                                                                                                                                                                            |
+| **date_of_birth**                                                                                                                                                                                                                                |           optional           | *String*  |                          |                                DD/MM/YYYY; MM/DD/YYYY(USA)                                | User's date of birth                                                                                                                                                                                                                                                                                                                  |
+| **country_of_residence**                                                                                                                                                                                                                         |           optional           | *String*  |                          |                                    code of the country                                    | User's country of residence                                                                                                                                                                                                                                                                                                           |
```

Comment:
More of a note - it would be useful if we added the link to the country codes list as a reference. Something like this, the standard used in the shopfiy/address package: https://www.iso.org/obp/ui/#search (the same for the states)